events/cmd/sender/sender.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/cloudevents/sdk-go/pkg/cloudevents"
+	"github.com/cloudevents/sdk-go/pkg/cloudevents/client"
+	cloudeventsnats "github.com/cloudevents/sdk-go/pkg/cloudevents/transport/nats"
+	"github.com/cloudevents/sdk-go/pkg/cloudevents/types"
+	"github.com/google/uuid"
+	"github.com/kelseyhightower/envconfig"
+	"github.com/metno/S-ENDA-Prototype/events/internal/config"
+	"github.com/metno/S-ENDA-Prototype/events/pkg/datastructs"
+)
+
+func main() {
+	var env config.EnvConfig
+	if err := envconfig.Process("", &env); err != nil {
+		log.Printf("[ERROR] Failed to process env var: %s", err)
+		os.Exit(1)
+	}
+	os.Exit(_main(os.Args[1:], env))
+}
+
+// Simple holder for the sending sample.
+type Demo struct {
+	Message string
+	Source  url.URL
+	Target  url.URL
+
+	Client client.Client
+}
+
+func (d *Demo) Send(eventContext cloudevents.EventContext, i int) (context.Context, *cloudevents.Event, error) {
+	event := cloudevents.Event{
+		Context: eventContext,
+		Data: &datastructs.Example{
+			Sequence: i,
+			Message:  d.Message,
+		},
+	}
+	return d.Client.Send(context.Background(), event)
+}
+
+func _main(args []string, env config.EnvConfig) int {
```

Comment:
Same naming thing here. Would be nice with function name reflecting what it does.